tests/compute/half-texture.slang.1.expected

```diff
@@ -0,0 +1,51 @@
+result code = 0
```

Comment:
This output is going to be very brittle, since anything that changes how we emit HLSL could conceivably break this test.